documentation/modules/auxiliary/scanner/http/billquick_txtid_sqli.md

```diff
@@ -0,0 +1,92 @@
+## Vulnerable Application
+
+This module exploits a SQL injection vulnerability in BillQUick Web 
+Suite prior to version 22.0.9.1. The application is .net based, and 
+the database is required to be MSSQL. Luckily the website gives 
+error based SQLi messages, so it is trivial to pull data from the 
+database. However the webapp uses an unknown password security 
+algorithm. This vulnerability does not seem to support stacked 
+queries. This module pulls the database name, banner, user, 
+hostname, and the SecurityTable (user table).
+
+### Install
+
+This install can be rather complicated and take about 2hrs to install.
+
+1. Download [ws2020](https://billquick.net/download/WS2020/WS2020Setup.zip)
+1. Download [Bill Quick 2020](https://billquick.net/download/Billquick2020/BillQuick2020Setup.zip)
+1. Install billquick 2020
+1. reboot
+1. Install IIS per WS2020 instructions (non-default options in ws2020 install docs)
+1. Install .NET Framework 3.5 (for sql server 2008, powershell: `Install-WindowsFeature Net-Framework-Core`)
+1. Install MSSQL Server 2008
+1. Install ws2020 (.NET 4.5 is bundled, may need a reboot)
+1. Open BillQuick V21 (on desktop). Configure it to a new database
+1. visit http://<ip>/ws2020 and finish the install/config
+
+Even at this point, 2 people with these instructions and one independently were unable to login to
+the webapp at this point.  It can be SQLi, but no one was able to use it successfully.
```

Comment:
Computers are awesome!

Reply:
Double 'at this point', needs to be reworded